be/src/exprs/bitmap_functions.cpp

```diff
@@ -74,8 +74,7 @@ StatusOr<ColumnPtr> BitmapFunctions::to_bitmap(FunctionContext* context, const s
             }
         }
 
-        BitmapValue bitmap;
-        bitmap.add(value);
+        BitmapValue bitmap(value);
 
         builder.append(&bitmap);
```

Comment:
why not call builder.append(std::move(bitmap));

Reply:
```
    void ColumnBuilder::append(const DatumType& value) {
        _null_column->append(DATUM_NOT_NULL);
        _column->append(value);
    }

    void ColumnBuilder::append(MovableType value) {
        _null_column->append(DATUM_NOT_NULL);
        _column->append(std::move(value));
    }

template <typename T>
void ObjectColumn<T>::append(T&& object) {
    _pool.emplace_back(std::move(object));
    _cache_ok = false;
}
```

Reply:
According to the actual test results, using std::move is slower, probably because it moves twice

Reply:
```
BitmapValue::BitmapValue(BitmapValue&& other) noexcept
        : _bitmap(std::move(other._bitmap)), _set(std::move(other._set)), _sv(other._sv), _type(other._type) {
    //other._sv = 0;
    //other._type = EMPTY;  // the two set will make move constructor slowly
}
```